test/EndToEnd/ProjectBuildTests.cs

```diff
@@ -96,13 +96,15 @@ public void ItCanRunAnAppUsingTheWebSdk()
                 .Should().Pass().And.HaveStdOutContaining("Hello, World!");
         }
 
-        [WindowsOnlyFact]
-        public void ItCanPublishArm64Winforms()
+        [WindowsOnlyTheory]
+        [InlineData("net5.0")]
+        [InlineData("net6.0")]
```

Comment:
I think we should probably have a test case where we don't explicitly specify the target framework, so that it will roll forward when the templates do.  I don't know if we need to explicitly test .NET 6, testing .NET 5 and the default (currently .NET 7) may be good enough.